components/config-mgmt-service/grpcserver/actions_handler.go

```diff
@@ -0,0 +1,53 @@
+package grpcserver
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/chef/automate/api/interservice/cfgmgmt/request"
+	"github.com/chef/automate/api/interservice/cfgmgmt/response"
+	"github.com/chef/automate/components/config-mgmt-service/backend/postgres"
+)
+
+func (s *CfgMgmtServer) HandlePolicyUpdateAction(ctx context.Context, req *request.PolicyUpdateAction) (*response.PolicyUpdateAction, error) {
+	// FIXME remove this or make it debug w/e
+	log.Infof("got HandlePolicyUpdateAction call %+v", *req)
```

Comment:
You have a note to remove this line

Reply:
haha, I looked for TODOs before I pushed